### PR TITLE
Fix importing a connector

### PR DESCRIPTION
**Pull request by vmanilo**
Github Issue: https://github.com/Twingate/terraform-provider-twingate/issues/94

## Changes
- Fixed connector import - set network_id

### CODE REVIEW

File: docs/resources/connector.md [hunk 1]
```diff
@@ -47,5 +47,6 @@ resource "twingate_connector" "aws_connector" {
 Import is supported using the following syntax:
 
 ```shell
+terraform import twingate_remote_network.aws_network UmVtb3RlTmV0d29yazozOTkwNw==
```
Comment (romankor) [inline docs/resources/connector.md:50]: i don't think we need to add these imports to documentation, its kinda obvious that a network has to be present in order for the connector to work

File: examples/resources/twingate_connector/import.sh [hunk 2]
```diff
@@ -1 +1,2 @@
-terraform import twingate_connector.connector Q29ubmVjdG1b0qe0
\ No newline at end of file
+terraform import twingate_remote_network.aws_network UmVtb3RlTmV0d29yazozOTkwNw==
```
Comment (romankor) [inline examples/resources/twingate_connector/import.sh:1]: same here

**Review by romankor — CHANGES_REQUESTED** [review]
Fix .. remove imports from documentation, but I verified the fix and it works

File: docs/resources/connector.md (hunk 1 above)
Comment (vmanilo) [reply docs/resources/connector.md]: removed it

File: examples/resources/twingate_connector/import.sh (hunk 2 above)
Comment (vmanilo) [reply examples/resources/twingate_connector/import.sh]: removed it

**Review by romankor — APPROVED** [review]
GTM

**Pull request merged**